### PR TITLE
D1b: allow inspection with hands

### PR DESCRIPTION
**Pull request by Laura-O**
Nobody complained in #460, so I am going ahead with my idea:
I changed D1b so that competitors must use their feet only after starting the timer, so inspection with hands is allowed.

As far as I can see, this change doesn't generate any loopholes:
- The inspection starts like in any other speedsolving event, competitors can pick up their cube or leave it on the mat and inspect with their feet like they could do in any other event.
- Starting the timer is defined in D3 and has to be done by placing the feet on the timer.
- Stopping is in D4.

### CODE REVIEW

**Comment by lgarron** [comment]
I personally still don't like this change. I still think OH should have one-handed inspection. :-P
However, I think it's the right thing to do.

Could you also include a guideline similar to https://www.worldcubeassociation.org/regulations/guidelines.html#3h2+ ? Suggestion:

> D1b++) CLARIFICATION In the past, competitors were required to inspect the puzzle using their feet for Solving with Feet. They are now permitted to touch the puzzle with their hands or any other part of their body during inspection (see Guideline A5b+).

**Comment by Laura-O** [comment]
I added the guideline, that's a good idea!

Lesson learned: if you want to use the [wrc](https://github.com/thewca/wca-regulations-compiler) and you installed the "Wine Resource Compiler" you might get weird error messages. :-(

File: wca-regulations.md [hunk 1]
```diff
@@ -467,7 +467,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 
 - D1) Standard speed solving procedures are followed, as described in [Article A](regulations:article:A) (Speed Solving). Additional regulations that supersede the corresponding procedures in [Article A](regulations:article:A) are described below.
     - D1a) During the attempt, the competitor must sit in a chair, sit on the surface, or stand.
-    - D1b) During the attempt, the competitor must use only their feet and the surface to manipulate the puzzle. Penalty: disqualification of the attempt (DNF).
+    - D1b) After starting the timer, the competitor must use only their feet and the surface to manipulate the puzzle. Penalty: disqualification of the attempt (DNF).
```
Comment (AlbertoPdRF) [inline wca-regulations.md]: For consistency with [C1b)](https://www.worldcubeassociation.org/regulations/#C1b) I would change

> After starting the timer,

for

> During the solve,

Comment (Laura-O) [reply wca-regulations.md]: Good catch. I can't remember why I changed this, but your suggestion is definitely better.

**Comment by AlbertoPdRF** [comment]
LGTM!

**Comment by kingmathyall** [comment]
LGTM too!

**Pull request merged**